app/assets/javascripts/application.js

```diff
@@ -14,4 +14,4 @@
 //= require popper
 //= require bootstrap-sprockets
 //= require activestorage
-//= require_tree .
```

Comment:
Did you test how this deletion influences on user interface?

Reply:
))))